### ISSUE TITLE
Portainer Business Edition Let's Encrypt Certificates

### ISSUE THREAD

**Issue by Morpheus8** [MikeJMcGuire/HASSAddons#131]
Thanks for this great Addon, I love it!

It would be great if you could use the Let's Encrypt certificate from Home Assistant for the Portainer EE addon. Same as with other addons, where you can specify the path to the certificate in the settings for the addon.

Example from a other Addon:
![image](https://github.com/MikeJMcGuire/HASSAddons/assets/44366448/36922035-c8d5-4ffb-9d34-35004a320a37)

**Comment by MikeJMcGuire** [comment]
Will have a look.

**Comment by MikeJMcGuire** [comment]
This is for when you're not using it via ingress?

**Comment by Morpheus8** [comment]
Yes that would be great, exactly the certificate would only be used if I connect to Portainer directly over the open Port and not via ingress.
